dist/main.js

```diff
@@ -0,0 +1,28 @@
+// calsses init 
+class Main{
+    constructor(){
+        this.model = new Model()
+        this.view = new View()
+    }
+
+    updatePlayers(teamName){
+        this.model.getPlayers(teamName).then((value => this.view.render(value)))
+    }
+}
+
+
+const main = new Main() 
+// events 
+
+$(`.btn`).on('click',function(){
```

Comment:
If you are creating a `Main` class (which there is no need for) then you should put your event listeners in there as well...